packages/legacy/src/react-to-webcomponent.ts

```diff
@@ -0,0 +1,101 @@
+import type {
+  ComponentClass,
+  Container,
+  FC,
+  R2WCOptions,
+  ReactElement,
+  RefObject,
+  Context,
+} from "@r2wc/core"
+import r2wcCore from "@r2wc/core"
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+const rootSymbol = Symbol.for("r2wc.root")
+
+interface ReactDOMType {
```

Comment:
are these the same types from the legacy code?

Reply:
yes they are